docker-compose.yml

```diff
@@ -35,14 +35,16 @@ x-sentry-defaults: &sentry_defaults
     PYTHONUSERBASE: "/data/custom-packages"
     SENTRY_CONF: "/etc/sentry"
     SNUBA: "http://snuba-api:1218"
+    REQUESTS_CA_BUNDLE: "/etc/ssl/certs/ca-certificates.crt"
+    GRPC_DEFAULT_SSL_ROOTS_FILE_PATH_ENV_VAR: "/etc/ssl/certs/ca-certificates.crt"
```

Comment:
Why not `DEFAULT_CA_BUNDLE`? Why was that in the original and why do we not need it here?

Reply:
That was defined as a "common variable" to not repeat the value, since we cannot do that here elegantly, opted to axe it and repeat the value.

Reply:
@BYK Actually, no. botocore uses it.

Reply:
Thanks for speaking up @Spice-King, I had a hunch but didn't follow it up.

Reply:
Ah, great then I'll bring it back.

Reply:
I'd not have used it if it was not needed, but it was a more descriptive one to try and be DRY with, so I used to define the rest.

Reply:
We should document this, at least in the code here, possibly on `develop`.

Reply:
Done and done.